mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -48,6 +49,10 @@
  */
 public interface MailboxListener {
 
+    interface GroupMailboxListener extends MailboxListener {
```

Comment:
Group is an element in EventBus system, you defined a new kind of Listener relating to Group in MailboxListener API level. I see the mixing between two API levels.

Then you switched existing listeners to GroupMailboxListener. What I understand is you tighten existing listeners to the EventBus system.

Reply:
`Group is an element in EventBus system` No it is defined in `mailbox/api`

Technically, we can extract `GroupMailboxListener` to a standalone class. But for me, it is a concept that makes perfect sens: A listener that ships its default group along.

Reply:
After discussing this, we came to an agreement that we should move `Event` and `MailboxListener` to the `event` package of the mailbox api, in a later PR at the end of the sprint to minimise conflicts.